code/modules/research/rdconsole.dm

```diff
@@ -351,6 +352,7 @@ cause a ton of data to be lost, an admin can go send it back.
 		printer_reagent_list += list(list(
 			"name" =           R.name,
 			"volume" =         R.volume,
+			"type" = R,
```

Comment:
a datum isn't a typepath, this seems odd

Reply:
R.type didn't work. Nothing happened when I tried to purge one of the chems